qemu/tests/balloon_check.py

```diff
@@ -119,6 +119,7 @@ def _memory_stats_compare(self, keyname, memory_stat_qmp):
         :param memory_stat_qmp: memory stat values from qmp.
         """
         check_mem_ratio = float(self.params.get("check_mem_ratio", 0.1))
+        check_mem_diff = float(self.params.get("check_mem_diff", 150))
```

Comment:
The logic is ok for me,  but not very sure about the acceptable value here,  wait for @lijinlijin's idea.